src/js/view/ipcListener.js

```diff
@@ -9,18 +13,18 @@ export const ipcListener = (dispatch, state) => {
           type: action.type,
           data: action.data
         });
-        let argObj = {};
-        argObj.filePath = action.data[0];
-        argObj.numberOfLines = 5;
-        argObj.lineNumber = 10;
-
-        argObj.function = 'liveLines';
-        sendRequestToBackend(argObj);
-        argObj.function = 'numberOfLines';
-        sendRequestToBackend(argObj);
-        argObj.function = 'fileSize';
-        sendRequestToBackend(argObj);
-
+        initializeOpenFile(action.data[0]);
+        break;
+      case 'saveState':
+        saveStateToDisk();
+        break;
+      case 'loadState':
+        populateStore(JSON.parse(action.data));
+        break;
+      case 'backendError':
+        //handle errors in the future
```

Comment:
I think it could be better to put up an issue to fix this and just skip the comments.